src/syscall/syscall_libc_darwin.go

```diff
@@ -168,10 +173,10 @@ type Stat_t struct {
 	Gid       uint32
 	Rdev      int32
 	Pad_cgo_0 [4]byte
-	Atim      Timespec
-	Mtim      Timespec
-	Ctim      Timespec
-	Btim      Timespec
+	Atimespec Timespec
```

Comment:
Probably want to remove the misleading comment above, mea culpa.

Reply:
Good point. I'll remove the comment.